charts/enterprise-logs/exports/default/apps-v1.Deployment-enterprise-metrics-admin-api.yaml

```diff
@@ -0,0 +1,121 @@
+apiVersion: apps/v1
```

Comment:
Also related to the `exports` discussion in the Makefile, I'd consider removing the entire `exports` directory.